### DISCUSSION TITLE
Imposing a Standard Deviation Restriction within Intervals

### DISCUSSION THREAD

**Discussion by JoaoPasseri** [guillermo-navas-palencia/optbinning#322] [General]
Hello. Would it be possible to impose a restriction on the standard deviation within each interval? For example, each interval should have a maximum std of 0.05.

**Comment by guillermo-navas-palencia** [comment]
Yes, that constraint is doable. Why do you think this constraint would be useful for a greater audience?

**Reply by JoaoPasseri** [reply]
I believe this could be helpful if I want the behavior of the groups, defined by the intervals, to remain consistent over time.

For example, suppose I have SCORE data for contracts over a one-year period and apply the optbinning technique. When analyzing the monthly average default rate for each interval, the curves for each group might intersect within that one-year timeframe. However, my goal would be for these curves not to intersect within that same one-year period, maintaining consistency and separation.
